scraper.rb

```diff
@@ -5,13 +5,15 @@
 require 'pry'
 require 'require_all'
 require 'scraped_page_archive/open-uri'
+require 'scraperwiki'
 
 require_rel 'lib'
 
-page = MembersPage.new(
-  response: Scraped::Request.new(
-    url: 'http://house.gov.by/ru/deputies-ru/'
-  ).response
-)
-
-puts page.member_urls
+MembersPage.new(
+  response: Scraped::Request.new(url: 'http://house.gov.by/ru/deputies-ru/').response
+).member_urls.each do |member_url|
+  response = Scraped::Request.new(url: member_url).response
+  data = MemberPage.new(response: response).to_h.merge(term_id: 6)
+  puts data
```

Comment:
this will introduce lots of output, and make Morph unhappy. As you only introduced it in https://github.com/everypolitician-scrapers/belarus-house-of-representatives-2016/pull/6/commits/89fe6617cab4d2211b97124ed71fbd63264314cb, I'm assuming it was an accidental inclusion from a local test run.